projects/ixocreate/ngx-admin/src/lib/pipes/safe-url.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({
+  name: 'safeUrl'
+})
+export class SafeUrlPipe implements PipeTransform {
```

Comment:
If you add a link in the src part of a html component, you need to sanitize it. This is a "| safeUrl" pipe to correct that automatically

Reply:
Makes sense - sounds fine to me in this case. Let's make sure to use it with care :)